src/deltacode/__init__.py

```diff
@@ -80,6 +82,21 @@ def align_scans(self):
             for f in self.old.files:
                 f.original_path = f.path
 
+    def similarity(self):
+        for delta in self.deltas:
+            if delta.new_file == None or delta.old_file == None:
+                continue
+            new_fingerprint = delta.new_file.fingerprint
+            old_fingerprint = delta.old_file.fingerprint
+            if new_fingerprint == None or old_fingerprint == None:
+                continue
+            new_fingerprint = utils.bitarray_from_hex(delta.new_file.fingerprint)
+            old_fingerprint = utils.bitarray_from_hex(delta.old_file.fingerprint)
+            hamming_distance = utils.hamming_distance(new_fingerprint, old_fingerprint)
+            delta.score += hamming_distance
```

Comment:
We should probably move this line down below into the `if hamming_distance <= similarity_threshold`